phys/module_surface_driver.F

```diff
@@ -1480,6 +1501,12 @@ SUBROUTINE surface_driver(                                         &
          RAINBL(i,j) = RAINBL(i,j) + RAINCV(i,j) + RAINNCV(i,j)
          IF ( PRESENT( rainshv ))RAINBL(i,j) = RAINBL(i,j) + RAINSHV(i,j)
          RAINBL(i,j) = MAX (RAINBL(i,j), 0.0)
+         sf_surf_irr: SELECT CASE(sf_surf_irr_scheme)
+            CASE(DRIP)
+               CALL drip_irrigation(julian_in,IRRIGATION(i,j),sf_surf_irr_scheme, irr_daily_amount,irr_start_hour,irr_num_hours,irr_start_julianday,irr_freq,irr_ph,i,j,RAINBL(i,j),IRRIGATION_CHANNEL(i,j),gmt,xtime,dt,irr_rand_field(i,j))
```

Comment:
@arjanna 
Arianna,
Use continuation "&" characters for lines over the Fortran standard.